### ISSUE TITLE
Resolution of ModuleArg is broken in some commands

### ISSUE THREAD

**Issue by odisseus** [propensive/fury-old#1246]
## Steps to reproduce
```
fury layer init 
fury project add -n scala
fury module add -n compiler -t compiler
fury module update -C scala-lang.org:scala-compiler:2.12.8
```
### Expected result
All commands succeed
### Actual result
```
The module has not been specified.
```
## Stack trace
```
fury.strings.Unspecified
    at fury.strings.Unspecified$.apply(strings.scala:55)
    at fury.strings.package$OptionExtensions.asTry(strings.scala:46)
    at fury.ModuleCli.$anonfun$update$31(module.scala:185)
    at scala.util.Success.flatMap(Try.scala:251)
    at fury.ModuleCli.$anonfun$update$30(module.scala:184)
    at scala.util.Success.flatMap(Try.scala:251)
    at fury.ModuleCli.$anonfun$update$29(module.scala:183)
    at scala.util.Success.flatMap(Try.scala:251)
    at fury.ModuleCli.$anonfun$update$28(module.scala:182)
    at scala.util.Success.flatMap(Try.scala:251)
    at fury.ModuleCli.$anonfun$update$23(module.scala:171)
    at scala.util.Success.flatMap(Try.scala:251)
    at fury.ModuleCli.$anonfun$update$20(module.scala:169)
```

This problem may be related to the pull request #1171.

**Comment by propensive** [comment]
I can't repeat this with the latest master.

**Comment by odisseus** [comment]
This issue was actually caused by #1247. Fixed by pull request #1248.
